### PR TITLE
process header for core idea collection

### PR DESCRIPTION
**Pull request by der-john**
(needs #2573.) Part of the redesign.

! This doesn't do anything unless you set `hasResourceHeader` to `true` in `src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Process.ts`, line 85.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html [hunk 1]
```diff
@@ -1,8 +1,44 @@
...
     <div class="action-bar">
-        <adh-print-action data-class="action-bar-item"></adh-print-action>
         <adh-map-switch data-ng-if="processProperties.hasLocation" data-model="data.isShowMap"></adh-map-switch>
     </div>
+</div>
+<div class="action-bar" data-ng-if="!hasResourceHeader">
+    <adh-print-action data-class="action-bar-item"></adh-print-action>
+    <adh-map-switch data-ng-if="processProperties.hasLocation" data-model="data.isShowMap"></adh-map-switch>
+</div>
+<div>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html]: Why did you move this functionality out of the wrapper div?

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+                <div class="resource-header-main">
+
+                    <h1 class="resource-header-title">{{ data.title }}</h1>
+
+                    <ul class="resource-header-meta">
+                        <li>
+                            {{ "TR__BY" | translate }} <adh-user-meta data-path="{{data.creator}}" data-ng-if="data.creator"></adh-user-meta>
+                        </li>
+                        <li>
+                            {{ "TR__ON" | translate }} <adh-time data-datetime="data.creationDate" data-format="L"></adh-time>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html]: `data.creationDate` is not defined.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Process.ts [hunk 3]
```diff
@@ -53,6 +58,9 @@ export var detailDirective = (
                         var sheet = resource.data[SIWorkflow.nick];
                         var stateName = sheet.workflow_state;
                         scope.currentPhase = AdhProcess.getStateData(sheet, stateName);
+                        scope.data.title = resource.data[SITitle.nick].title;
+                        scope.data.creator = resource.data[SIMetadata.nick].creator;
+                        scope.data.description = resource.data[SIDescription.nick].description;
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Process.ts]: IMHO we should use `short_description` here

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html [hunk 4]
```diff
@@ -1,8 +1,44 @@
...
+                        </li>
+                    </ul>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html]: I don't think that this meta data is very useful. How about we remove it for now and think about useful meta data separately?

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html [hunk 5]
```diff
@@ -1,8 +1,44 @@
...
+                    <ul class="resource-header-meta">
+                        <li>
+                            <i class="icon-pin-detail"></i> {{data.description}}
+                        </li>
+                    </ul>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html]: `resource-meta-data` has uppercase styling by default. We should have a separate class for description.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html (hunk 4 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html]: good idea.

**Comment by der-john** [comment]
is being rebased.
